docs/migration-guide-2.9-2.10.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Migration Guide 2.9-2.10
+---
+
+Due to the removal of `kyma-integration` namespace, we need to migrate all Application Connector secrets to the `kyma-system` namespace. After upgrading the Kyma to the 2.10 version, you need to execute the script:
+
+For Kyma SKR - [2.9-2.10-SKR-copy-secrets-to-system-namespace.sh](./assets/2.9-2.10-SKR-copy-secrets-to-system-namespace.sh)
```

Comment:
@grego952 , please, check if this is a good place for such a note

Reply:
Nope, we need to provide a separate migration guide for SKR. Like [this one](https://help.sap.com/docs/BTP/922bf2dbe0b646aaaa8cb5e077cfd799/70470339bd09423782c18f2c3a5d7f28.html?locale=en-US).